client/src/components/MultiJurisdictionAudit/AASetup/Contests/ContestForm.tsx

```diff
@@ -77,26 +78,43 @@ const ContestForm: React.FC<IProps> = ({
   /* istanbul ignore next */
   if (isBatch && !isTargeted && nextStage.activate) nextStage.activate() // skip to next stage if on opportunistic contests screen and during a batch audit (until batch audits support multiple contests)
 
+  const initialValues = {
+    contests: filteredContests.length ? filteredContests : contestValues,
+  }
+
+  const isOpportunisticFormClean = (
+    touched: {},
+    values: { contests: IContest[] }
+  ) => {
+    return (
+      !isTargeted &&
+      isObjectEmpty(touched) &&
```

Comment:
Does this mean that if I click into a field and then click out of it without entering anything, it still won't let me move to the next screen? Could we be more permissive and let them skip this step as long as the actual data still equals the initial default empty values?

Reply:
@jonahkagan It does mean that, yeah. If we changed it to an `||` we could make it like you're saying. I agree with you that that would be better.